frontend/src/store/index.js

```diff
@@ -341,6 +342,48 @@ const getters = {
       return get(cloudProfile, 'data.providerConfig.constraints.loadBalancerConfig.classes')
     }
   },
+  partitionIDsByCloudProfileNameAndRegion (state, getters) {
+    return ({ cloudProfileName, region }) => {
+      // Partion IDs equal zones for metal infrastructure
+      const cloudProfile = getters.cloudProfileByName(cloudProfileName)
+      if (cloudProfile.metadata.cloudProviderKind !== 'metal') {
```

Comment:
can we be sure that cloudProfile / cloudProfile.metadata. is always there? same applies for methods introduced below

Reply:
I would say yes if we don't have an incorrect state?

Reply:
implemented safe access